data/mapcss-mapping.csv

```diff
@@ -1351,3 +1351,5 @@ recycling|shoes;[recycling:shoes=yes];;name;int_name;1352;
 recycling|green_waste;[recycling:green_waste=yes];;name;int_name;1353;
 recycling|cartons;[recycling:cartons=yes];;name;int_name;1354;
 amenity|parking_space|disabled;[amenity=parking_space][parking_space=disabled];;name;int_name;1355;
+amenity|music_school;1356;
+amenity|language_school;1357;
```

Comment:
nit: Put this definitions somewhere instead of "deprecated." Lets say into 1148, 1149 near "amenity|driving_school;1161;"